.github/workflows/release.yml

```diff
@@ -12,28 +12,6 @@ on:
       version:
         description: 'The version to release (e.g. 1.2.3). This workflow will automatically perform the required version bumps'
         required: true
-      update_changelog:
```

Comment:
What's the proposal for this?

Reply:
That's a good point. Let me try to reintroduce some of them where possible or relevant.